packages/schedule/src/ScheduleContainer.spec.js

```diff
@@ -15,89 +16,114 @@ import ScheduleContainer from './ScheduleContainer';
 jest.useFakeTimers();
 
 describe('ScheduleContainer', () => {
-  let wrapper;
   let requestAnimationFrame;
   let cancelAnimationFrame;
 
   const mockRaf = createMockRaf();
 
-  const basicExample = props => (
-    <ScheduleContainer {...props}>{elapsed => <p>{elapsed}</p>}</ScheduleContainer>
+  const BasicExample = props => (
+    <ScheduleContainer {...props}>
+      {elapsed => <p data-test-id="schedule">{elapsed}</p>}
+    </ScheduleContainer>
   );
 
   beforeEach(() => {
     requestAnimationFrame = jest
       .spyOn(window, 'requestAnimationFrame')
       .mockImplementation(mockRaf.raf);
     cancelAnimationFrame = jest.spyOn(window, 'cancelAnimationFrame');
-    setTimeout.mockClear();
+    clearTimeout.mockClear();
   });
 
   afterEach(() => {
     window.requestAnimationFrame.mockRestore();
     jest.clearAllTimers();
+    jest.clearAllMocks();
   });
 
   it('sets up requestAnimationFrame', () => {
-    wrapper = mount(basicExample());
+    render(<BasicExample />);
+
+    act(() => {
```

Comment:
Is this `act` necessary? The docs show it's usage wrapping a `render` normally https://reactjs.org/docs/test-utils.html#act

Reply:
for the initial render it is fine but any state updates from the rAF needs to be wrapped in `act`. I've updated the tests so only the ones that need it are wrapped.